tests/x11/thunderbird/thunderbird_install.pm

```diff
@@ -0,0 +1,38 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Thunderbird installation
+# Maintainer: Paolo Stivanin <pstivanin@suse.com>
+
+use base "x11test";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use version_utils qw(is_leap);
+
+sub run {
+    my ($self) = @_;
+
+    select_console 'root-console';
+
+    pkcon_quit;
+    zypper_call("ar -f http://download.opensuse.org/distribution/leap/" . get_var('VERSION') . "/repo/oss repo-oss-tmp") if is_leap;
```

Comment:
Is this really needed on Leap? Just wondering why is not oss repo enabled by default.

Reply:
Mmm no, it's not needed. I added it because no repos were being added to my qcow2 image, but I think it's because I forgot to add some env vars when creating the job (http://d502.qam.suse.de/tests/1346#settings)